InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -117,6 +121,88 @@ public void robotInit() {
         lastTime = System.currentTimeMillis();
     }
 
+    public void wireUpButtons() {
+
+        Buttons b = new Buttons();
+
+        //////////////////////////////////////////////
+        // Drive Subsystem
+        if (config.enableDriveSubsystem) {
+            b.driveAutoAim.whenHeld(new InstantCommand(() -> driveSubsystem.setAutoAligning(true), driveSubsystem)).whenReleased(new InstantCommand(() -> driveSubsystem.setAutoAligning(false), driveSubsystem));
+
+            driveSubsystem.setDefaultCommand(new RunCommand(
+                () -> driveSubsystem.drive(
+                    b.driverControl.getRawAxis(b.driveSpeed),
+                    b.driverControl.getRawAxis(b.driveTurn)),
+                driveSubsystem)
+            );
+        }
+
+        //////////////////////////////////////////////
+        // Intake Subsystem
+        if (config.enableIntakeSubsystem) {
+            b.operatorToggleIntake.whenPressed(new InstantCommand(() -> intakeSubsystem.toggleIntakeArm(), intakeSubsystem));
+
+            b.operatorIntake.whenHeld(new InstantCommand(() -> intakeSubsystem.intake(), intakeSubsystem)).whenReleased(new InstantCommand(() -> intakeSubsystem.off(), intakeSubsystem));
+            b.operatorOutake.whenHeld(new InstantCommand(() -> intakeSubsystem.outake(), intakeSubsystem)).whenReleased(new InstantCommand(() -> intakeSubsystem.off(), intakeSubsystem));

```

Comment:
For the sake of our laptop using students, we should probably split the whenHeld and whenReleased events to different lines.

Reply:
forgot you could do that